PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,10 +5,11 @@
 - make the pull request against the ["v0.6" branch](https://github.com/Bitmessage/PyBitmessage/tree/v0.6)
 - it should be possible to do a fast-forward merge of the pull requests
 - PGP-sign the commits included in the pull request
+- try to use a good editor that removes trailing whitespace, highlights potential python issues and uses unix line endings
 - You can get paid for merged commits if you register at [Tip4Commit](https://tip4commit.com/github/Bitmessage/PyBitmessage)
 
 If for some reason you don't want to use github, you can submit the patch using Bitmessage to the "bitmessage" chan, or to one of the developers.
 ## Translations
 
 For helping with translations, please use [Transifex](https://www.transifex.com/bitmessage-project/pybitmessage/). There is no need to submit pull requests for translations.
-For translating technical terms it is recommended to consult the [Microsoft Language Portal](https://www.microsoft.com/Language/en-US/Default.aspx).
\ No newline at end of file
+For translating technical terms it is recommended to consult the [Microsoft Language Portal](https://www.microsoft.com/Language/en-US/Default.aspx).
```

Comment:
I must be getting blind but can't spot the change in this line.

Reply:
Toggling 'whitespace' in the github diff settings hides the line. This line came through in the email notification of your comment (but I can't find it in the github UI):

> \ No newline at end of file